policyengine_us/parameters/gov/states/vt/tax/income/credits/cdcc/low_income/income_threshold.yaml

```
@@ -0,0 +1,22 @@
+description: Vermont limits the low-income child and dependent care credit to filers with adjusted gross income up to this amount.
+metadata:
+  reference:
+    - title: 2021 Form IN-112 Vermont Tax Adjustments and Credits
+      href: https://tax.vermont.gov/sites/tax/files/documents/IN-112-2021.pdf#page=2
+    - title: 2021 Vermont Statutes, Title 32 - Taxation and Finance, Chapter 151 - Income Taxes, §5828c. Low-income child and dependent care credit
+      href: https://law.justia.com/codes/vermont/2021/title-32/chapter-151/section-5828c/
+  unit: currency-USD
+  period: year
+  label: Vermont low-income child and dependent care credit agi threshold
+  breakdown:
+  - filing_status
+JOINT:
+  2021-01-01: 40_000
```

And elsewhere, given this:
>(Added 2001, No. 144 (Adj. Sess.), § 24, eff. June 21, 2002; amended 2003, No. 70 (Adj. Sess.), § 39, eff. March 1, 2004.)
```suggestion
  2005-01-01: 40_000
```